### PR TITLE
app-shells/radian: add ebuild

### PR DESCRIPTION
**Pull request by mschubert**
A modern R console

### CODE REVIEW

File: app-shells/radian/radian-0.6.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+	dev-python/prompt-toolkit[$PYTHON_USEDEP]
+	dev-python/rchitect[$PYTHON_USEDEP]
+	dev-python/six[$PYTHON_USEDEP]
+	dev-lang/R"
```
Comment (AndrewAmmerlaan) [inline app-shells/radian/radian-0.6.9.ebuild:23]: These are probably `RDEPEND`s.

DEPEND is for things we need present during build
BDEPEND is for things we need to execute in order to build
RDEPEND is for things we need during runtime of what we have built

Comment (mschubert) [reply app-shells/radian/radian-0.6.9.ebuild:26]: They can indeed be `REPEND`s

**Comment by mschubert** [comment]
This should be good now

**Comment by AndrewAmmerlaan** [comment]
Sorry, I just noticed, but we have what looks to be the same package already in `::guru` but as `sci-mathematics/radian`. 

I'd like to avoid duplication (this only serves to double work), but the version in `::guru` is older, so maybe you can push this new version to `::guru`?

**Pull request closed without merging**